### ISSUE TITLE
CVE-2024-5932 

### ISSUE THREAD

**Issue by karkis3c** [projectdiscovery/nuclei-templates#10632]
The GiveWP – Donation Plugin and Fundraising Platform plugin for WordPress is vulnerable to PHP Object Injection in all versions up to, and including, 3.14.1 via deserialization of untrusted input from the 'give_title' parameter. This makes it possible for unauthenticated attackers to inject a PHP Object. The additional presence of a POP chain allows attackers to execute code remotely, and to delete arbitrary files.

**Template link**
https://raw.githubusercontent.com/karkis3c/bugbounty/main/nuclei-templates/cves/CVE-2024-5932.yaml

Thanks!
_-_ karkis3c

**Comment by DhiyaneshGeek** [comment]
Hi @karkis3c 

we already have a template for this [http/cves/2024/CVE-2024-5932.yaml](https://github.com/projectdiscovery/nuclei-templates/blob/main/http/cves/2024/CVE-2024-5932.yaml)

so we are closing this issue 

Thanks

**Comment by karkis3c** [comment]
Hello @DhiyaneshGeek,

Can you tell me the correct usage?

Do I need to pass the domain or the full URL of the donation link?

```
nuclei -u https://host
```

```
nuclei -u https://host/path/to/donation
```

Best,
@karkis3c

**Comment by DhiyaneshGeek** [comment]
Hi @karkis3c 

you can pass the domain name

```
nuclei -u https://host
```

If it doesn't work let me know 

Thanks

**Comment by karkis3c** [comment]
Tested not worked!

I reviewed the template and found that it doesn't work in all cases.

In my case, page have two slug key element:

i.e.,
_slug: "give-donation"_
_slug: "donation-form"_

The request proceeds as:

`GET /give/give-donation?giveDonationFormInIframe=1`

If the expected elements not found, the template fails. 

For my target. page doesn't have a placeholder value for the "_give-amount_" element, but it works when the second slug key element is used.

`GET /give/donation-form?giveDonationFormInIframe=1`

let me know where I can share the target

Thank you
